File: ide/src/main/java/org/dau/ide/IdeLauncher.java
package org.dau.ide;

import javafx.application.Application;
import org.dau.ide.logging.FxLogConfig;
import org.dau.ide.logging.FxLogManager;

public class IdeLauncher {

  public static void main(String... args) throws Exception {
    System.setProperty("java.util.logging.manager", FxLogManager.class.getName());
    System.setProperty("java.util.logging.config.class", FxLogConfig.class.getName());
    Application.launch(Ide.class, args);
  }
}

File: ide/src/main/kotlin/org/dau/ide/IdeLauncher.kt
package org.dau.ide

import javafx.application.Application
import org.dau.ide.logging.FxLogConfig
import org.dau.ide.logging.FxLogManager

object IdeLauncher {
  @JvmStatic
  fun main(args: Array<String>) {
    System.setProperty("java.util.logging.manager", FxLogManager::class.java.name)
    System.setProperty("java.util.logging.config.class", FxLogConfig::class.java.name)
    Application.launch(Ide::class.java, *args)
  }
}